modules/nodes-nodes-audit-log-advanced.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-advanced-{context}"]
+= Advanced Audit
```

Comment:
Do we have 2 options for auditing now? Or is it just one? 

If its just one call this 'auditing'!

Reply:
@sferich888 This is how the audit feature was presented in 3.x, Basic and Advanced. 
Is there someone who can confirm if the two are available in 4.x?